### PR TITLE
ui: adapt color badges

### LINKED ISSUE #213: badges background color do not match
## index/recentuploads
* **Resource type:**  uses color `teal` from semantic ui class.
* **Publication date:** uses a class name `record-version` which defines color `greyDark:#777`
recentuploads

## search & records landingpage
* **Resource type:** uses `grey` class of semantic ui.
* **Publication date:** uses `blue` class of semantic ui.
searchpage

### PR DESCRIPTION
**Pull request by rekt-hard**
closes #213

### CODE REVIEW

**Comment by rekt-hard** [comment]
Index Page
image


Search Page
image

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/frontpage.less [hunk 1]
```diff
@@ -20,9 +20,12 @@ h2 {
   margin-inline-end: 0px;
 }
 
+
+.ui.label.blue,
 .record-version {
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/frontpage.less:25]: remove the class recod-version
```suggestion
 {
```

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/frontpage.less [hunk 2]
```diff
@@ -20,9 +20,12 @@ h2 {
   margin-inline-end: 0px;
 }
 
+
+.ui.label.blue,
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/frontpage.less:24]: also target the blue.label
```suggestion
.ui.blue.labels .label, .ui.blue.label 
```

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/search.less [hunk 3]
```diff
@@ -26,3 +26,13 @@
 .identifiers {
   display: inline;
 }
+
+.ui.items > .item .extra.labels-actions {
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/search.less:30]: add a short comment for what its done
```suggestion
// align badges to the right
.ui.items > .item .extra.labels-actions {
```

**Review by mb-wali — CHANGES_REQUESTED** [review]

**Review by mb-wali — APPROVED** [review]

**Pull request merged**